src/utils/get-form.js

```diff
@@ -17,6 +17,9 @@ export function getFormStateKey(state, model, s = defaultStrategy, currentPath =
   let result = null;
 
   s.keys(state).some((key) => {
+    if (key === '') {
+      return false;
```

Comment:
Should we also issue a `console.warn` here to warn the user that empty keys in state are currently not supported?

Reply:
I'd put a warn in here. Very much an edge case, but it's good to be aware.